site/content/3.12/release-notes/version-3.12/whats-new-in-3-12.md

```diff
@@ -2085,6 +2085,40 @@ DB-Servers in a cluster has been added:
 |:------|:------------|
 | `arangodb_vocbase_transactions_lost_subordinates_total` | Counts the number of lost subordinate transactions on database servers. |
 
+### RocksDB upgrade
+
+<small>Introduced in: v3.12.6</small>
+
+The RocksDB library has been upgraded from version 7.2.0 to 9.5.0.
+
+As a result, you may see performance improvements while using slightly less
+resources especially for mixed workloads.
```

Comment:
I'm hesitant about adding specific numbers here (30-40% according to Kaveh)

Reply:
I would not since it is hard to predict these numbers from one workload to another